### DISCUSSION TITLE
[WS2122] Individual Projects Discussion

### DISCUSSION THREAD

**Discussion by changkun** [mimuc/gp-ws2021#30] [Discussion] [answered]
This thread is open for the discussion of individual projects.

**Comment by changkun** [comment]
The individual proposals are avaliable at:

- https://github.com/mimuc/gp/pull/58/files
- https://github.com/mimuc/gp/pull/59/files
- https://github.com/mimuc/gp/pull/62/files
- https://github.com/mimuc/gp/pull/64/files
- https://github.com/mimuc/gp/pull/65/files

Everyone is welcome to write a review to these proposals.

**Comment by alvaxgln** [comment]
Hey there ^^

I'm pretty confused right now, since i just noticed the deadline for "Intermediate Submission" ends today. I thought during last session that deadline was extended?

Also there is no entry in the syllabus for the session next Monday.

**Reply by changkun** [reply]
Sorry for the confusion. I think I didn't push the committed content since our last discussion. For technical reasons, I could not reach that computer at the moment. I will push a new commit to that.

**Comment by alvaxgln** [comment, marked as answer]
Since the project intermediate pull request is still open, I don't think I can open a new one for the same branch to submit the final project. But the project intermediate pull request should now contain the final state, so will that work as a submission for the final project?

**Reply by changkun** [reply]
You can. The purpose of intermediate submission was to create the pull request, it can naturally be turned into a final submission (by renaming the PR's title).
